documentation/modules/exploit/linux/http/logsign_exec.md

```diff
@@ -0,0 +1,78 @@
+## Vulnerable Application
+
+Download the vulnerable version of OVA or ISO file from following URL. I strongly suggest you to choose OVA.
```

Comment:
Yes, it's possible to download iso from vendor page as well but I haven't tried it against this module. I'll add it anyway